src/reducers/reducerUtil.js

```diff
@@ -44,6 +44,19 @@ function handlePagedAction(action, state) {
                 cursor: cursor
             });
         }
+        case ActionTypes.SINGLE: {
+            let _contents = state.items;
+            _contents = {
+                ..._contents,
```

Comment:
im a bit confused by this, why are the array brackets necessary on the following line?  we're just mapping keys to objects here right?  is that what prevents duplicates?